fixtures/component_fixtures/EC2/component.yaml

```diff
@@ -12,7 +12,11 @@ satisfies:
     system_key: CloudFoundry
     verification_key: UAA_Verification_1
   implementation_status: partial
-  narrative: Justification in narrative form
+  narrative:
+    - key: "a"
+      text: "Justification in narrative form A for CM-2"
+    - key: "b"
+      text: "Justification in narrative form B for CM-2"
```

Comment:
Putting the models in that repository, you mean? Opened https://github.com/opencontrol/schemas/issues/12 to discuss further.